xml/yast2_sw.xml

```diff
@@ -320,47 +320,49 @@
-      <guimenu>Search Online</guimenu> </menuchoice>.
+     The online search feature allows searching for packages across all
+     registered and unregistered modules and extensions.
     </para>
-    <informalfigure>
-     <mediaobject>
-      <textobject role="description">
-       <phrase>&yast; software manager <guimenu>Online Search</guimenu>
-        dialog</phrase>
-      </textobject>
-      <imageobject role="fo">
-       <imagedata fileref="yast2_sw_online.png" width="70%" format="PNG"/>
-      </imageobject>
-      <imageobject role="html">
-       <imagedata fileref="yast2_sw_online.png" width="65%" format="PNG"/>
-      </imageobject>
-     </mediaobject>
-    </informalfigure>
-    <para>
-     To perform a search, write a term in the <guimenu>Package
-     Name</guimenu> text box and click the <guimenu>Search</guimenu> button.
-     &yast; then contacts the &scc; and, after a few seconds, shows the results
-     in the table, including the module or extension each package belongs to.
-    </para>
-    <!-- FIXME cwickert 2020-06-26: Usability for selecting packages is really
-     bad and subject to criticism, see https://bugzilla.suse.com/1165913
-     Behavior will probably change, make sure the following is still correct.
-    -->
-    <para>
-     Select one or more packages for installation by clicking the
-     corresponding table row and pressing the <guimenu>Toggle
-     Selection</guimenu> button. Alternatively, you can just double-click on a
-     row. If the package belongs to a not activated module or extension, &yast;
-     asks for confirmation to activate it. Click <guimenu>Next</guimenu>,
-     review the changes, and install the packages.
-    </para>
-   </sect3>
+    <procedure>
+     <title>Searching software online</title>
+     <para>
+      To search for software packages online, perform the following steps:
+     </para>
+     <step>
+      <para>
+       Open the online search window with <menuchoice><guimenu>Extras</guimenu>
+        <guimenu>Search Online</guimenu></menuchoice>.
+      </para>
+     </step>
+     <step>
+      <para>
+       Enter a <guimenu>Package Name</guimenu> and press
+       <keycap function="enter"/> or click <guimenu>Search</guimenu>.
+       &yast; contacts the &scc; and shows the results in a table, including the
+       module or extension of each package. Select a package to see additional
+       details.
+      </para>
+     </step>
+     <step>
+      <para>
+       Select one or more packages for installation by clicking the
+       corresponding table row and <guimenu>Toggle Selection</guimenu>.
+       Alternatively, you can just double-click on a row. If the package belongs
+       to an unregistered module or extension, &yast; asks for confirmation to
+       register it.
+      </para>
+     </step>
+     <step>
+      <para>
+       Click <guimenu>Next</guimenu>, review the changes, and install the
+       packages.
+      </para>
```

Comment:
Suggestion to add result para

Reply:
I see your point, but I left it out on purpose. In step 4 of the procedure I'm already installing a package, but installing and removing packages is covered by the following section. Thus I made the last step as brief. Adding a result ("You have successfully installed a package") would anticipate the next section.

I agree that the structure of the chapter is not optimal. We currently have a section for searching software and anther for installing or removing packages. I think that installing and removing are two separate things, and searching probably belongs to installing. Nevertheless I don't want to rewrite/restructure everything now.